howdoi/arxiv.py

```diff
@@ -0,0 +1,56 @@
+import xml.etree.ElementTree as ET
+from urllib.parse import urlencode
+import requests
+
+
+ROOT_URL = 'http://export.arxiv.org/api/'
```

Comment:
Looks nice and clean. My only recommendation would be adding some API specific comments that will help a new contributor understand what the API is about.

Reply:
Thanks! Any more changes (even minor nitpicks), let me know 💯